### PR TITLE
lots of small README edits

### PR DESCRIPTION
**Pull request by SethTisue**
note the addition of @dwijnand, @som-snytt, and @bishabosha, y'all might want to review your entries

### CODE REVIEW

**Comment by SethTisue** [comment]
among the folks who aren't currently listed at the top but could be if they want: @joroKr21, @martijnhoekstra, @mkeskells, @eed3si9n, ...

**Comment by SethTisue** [comment]
@NthPortal can I interest you in adding some verbiage about fatal warnings, e.g. locally vs. in CI?

File: README.md [hunk 1]
```diff
@@ -120,16 +133,27 @@ We are grateful for the following OSS licenses:
...
+within an sbt session.  This will build and publish the Scala
+distribution to your local artifact repository and then switch sbt to
+use that version as its new `scalaVersion`.  You may then revert back
+with `reload`.  Note `restarrFull` will also write the STARR version
+to `buildcharacter.properties` so you can switch back to it with
+`restarr` without republishing (though incremental compilation will
+recompile from scratch, sadly.)
 
 For history on how the current scheme was arrived at, see
 https://groups.google.com/d/topic/scala-internals/gp5JsM1E0Fo/discussion.
```
Comment (NthPortal) [inline README.md:160]: How's this @SethTisue?

```suggestion
https://groups.google.com/d/topic/scala-internals/gp5JsM1E0Fo/discussion.

**Building with fatal warnings**: To make warnings in the project fatal (i.e. turn them into errors), run `set Global / fatalWarnings := true` in sbt (replace `Global` with the name of a module—such as `reflect`—to only make warnings fatal for that module). To disable fatal warnings again, either `reload` sbt, or run `set Global / fatalWarnings := false` (again, replace `Global` with the name of a module if you only enabled fatal warnings for that module). CI always has fatal warnings enabled.
```

**Comment by joroKr21** [comment]
Sure, I'm always happy to help with higher-kinded types, implicits and variance.

**Review by lrytz — APPROVED** [review]

**Pull request merged**